### PR TITLE
Update to latest secondary-watcher

### PR DESCRIPTION
**Pull request by rjmac**
This deinverts the dependency so that the secondary-watcher is just an
ordinary executable jar instead of a plugin.

I think the run/run-with-log-level thing is superfluous now (it can just be "run" since now there's no runscript being inherited from secondary-watcher?).  There are probably other things to be worked out since neither of the other two secondaries went smoothly into this world, too.

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -128,7 +128,7 @@ object Deps {
     "org.elasticsearch" % "elasticsearch" % "1.7.2"
   )
   lazy val secondary = Seq(
-    "com.socrata" %% "secondarylib" % "3.0.5" exclude("org.slf4j", "slf4j-log4j12")
+    "com.socrata" %% "secondarylib" % "3.0.14" exclude("org.slf4j", "slf4j-log4j12")
```
Comment (sokrahta) [inline project/Build.scala]: it's probably worth releasing secondarylib with at least a minor revision bump with this app/plugin inversion.

File: spandex-secondary/docker/ship.d/run-with-log-level [hunk 2]
```diff
@@ -5,12 +5,7 @@ export UUID=$(cat /proc/sys/kernel/random/uuid)
 
 /bin/env_parse ${SERVER_CONFIG}.j2
 
-if [ ! -f secondary.conf.j2 ]; then
-  echo "ERROR: Create an image based on this for your secondary that includes a secondary.conf.j2 and run that."
-  exit 1
-fi
-
-/bin/env_parse secondary.conf.j2
+/bin/env_parse ${SECONDARY_CONFIG}.j2
```
Comment (sokrahta) [inline spandex-secondary/docker/ship.d/run-with-log-level]: I agree in thinking this docker entry point is now superfluous.

File: project/Build.scala (hunk 1 above)
Comment (rjmac) [inline project/Build.scala]: I thought about it, but while it's a BIG change from an operational perspective, it's a tiny tiny change from a code perspective...

Also this is the last secondary to get updated, both of the others are already released the new way.

**Comment by sokrahta** [comment]
This now runs locally with `sbt spandex-secondary/run`
And the docker deploy in marathon can be simplified to just `run` instead of `run-with-log-level`

**Comment by aerust** [comment]
lgtm :boat:

**Comment by rlvoyer** [comment]
🐑

**Pull request merged**